build.fsx

```diff
@@ -348,6 +348,7 @@ Target "MultiNodeTests" (fun _ ->
 
 Target "MultiNodeTestsNetCore" (fun _ ->
     if not skipBuild.Value then
+        setEnvironVar "akka.cluster.assert" "on" // needed to enable assert invariants for Akka.Cluster
```

Comment:
Causes MNTR runs to blow up if the `Gossip` class does something it shouldn't

Reply:
Will this be picked up by Akka, or do we need to change it to `AKKA_CLUSTER_ASSERT`?

Reply:
Ah, I didn't realize it was picked up as is inside the code, but maybe we should follow our own environment variable naming standard by making this `AKKA_CLUSTER_ASSERT`?

Reply:
Yeah we should - I'll update on that on a subsequent PR. I need to tweak some of the timing settings for `StressSpec` since the build server is way underpowered compared to my development machine. I'll address that with this too.